inc/saiport.h

```diff
@@ -572,6 +587,15 @@ typedef enum _sai_port_attr_t
     SAI_PORT_ATTR_INTERNAL_LOOPBACK_MODE,
 
     /**
+     * @brief Forward Error Correction (FEC) control
+     *
+     * @type sai_port_fec_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_FEC_MODE_NONE
```

Comment:
Is SAI_PORT_ATTR_AUTO_NEG_MODE is true, then this FEC_MODE will no longer be applicable to the port? if yes, then we need add validonly meta data to specify that FEC_MODE is only valid when AN_MODE = false.